lib/killbill_client/models/subscription.rb

```diff
@@ -132,6 +132,55 @@ def update_bcd(user = nil, reason = nil, comment = nil, effective_from_date = ni
       end
 
 
+      #
+      # Block a Subscription
+      #
+      def block_subscription(subscription_id, state_name, service, block_change, block_entitlement, block_billing, requested_date = nil, user = nil, reason = nil, comment = nil, options = {})
```

Comment:
1. No need to pass a `subscription_id` (it's an instance method)
2. Let's rename it to `set_blocking_state` (symmetry with `account.rb` and `bundle.rb`)